.travis.yml

```diff
@@ -21,27 +21,29 @@ env:
         - SETUP_CMD='test'
         - TEST_MODE='normal'
         - ATOM_DATA_URL='https://github.com/tardis-sn/tardis-atomdata/raw/master/kurucz_cd23_chianti_H_He.h5.zip'
+        - PLASMA_NLTE_REFERENCE_URL='https://github.com/vg3095/tardis-refdata/raw/plasma-ref2/plasma_reference/plasma_nlte_reference.h5'
+        - PLASMA_LTE_REFERENCE_URL='https://github.com/vg3095/tardis-refdata/raw/plasma-ref2/plasma_reference/plasma_lte_reference.h5'
         - MINICONDA_URL='http://repo.continuum.io/miniconda/Miniconda-latest-Linux-x86_64.sh'
 
 matrix:
     include:
         - python: 2.7
           env:
             - COMPILER=gcc
-            - SETUP_CMD='test --args="--atomic-dataset=$HOME/kurucz_cd23_chianti_H_He.h5"'
+            - SETUP_CMD='test --args="--atomic-dataset=$HOME/kurucz_cd23_chianti_H_He.h5 --plasma-reference=$HOME/plasma_reference/"'
```

Comment:
@wkerzendorf Here I want to pass path for `plasma-reference` files, but travis doesn't recognise this option.
![image](https://user-images.githubusercontent.com/12808297/29015065-0009127a-7b6a-11e7-84b0-af92d06a05bf.png)
I have added this option in `conftest.py` and this option works fine in my laptop. 
So, what should I do here?